libc/src/__support/RPC/rpc.h

```diff
@@ -351,15 +361,24 @@ struct Client {
   LIBC_INLINE Client(uint32_t port_count, void *buffer)
       : process(port_count, buffer) {}
 
-  using Port = rpc::Port<false, Packet<gpu::LANE_SIZE>>;
+#if defined(LIBC_TARGET_ARCH_IS_AMDGPU)
+  // The AMDGPU wavefront size is only known at runtime. We indicate this to the
+  // process using a value of UINT32_MAX.
```

Comment:
this isn't really accurate. it's known at compile time of the kernel

Reply:
Compile time has a slightly different meaning in terms of C++, it more or less means it's not `constexpr`. However, it's definitely not correct to say it's a runtime value.